data/Piranha.Data.EF/Db.cs

```diff
@@ -155,7 +155,7 @@ public Db(DbContextOptions<Db> options) : base(options)
                     if (!IsInitialized)
                     {
                         // Migrate database
-                        Database.Migrate();
+                        //Database.Migrate();
```

Comment:
why did you do this?

Reply:
> why did you do this?

I showed  you that bug, so decided to comment

Reply:
you mean not working, okay, we need to solve it - will add to todo list

Reply:
please add todo comment there, like
//TODO: fix migrations